composer.json

```diff
@@ -18,7 +18,9 @@
     "php-http/message-factory": "^1.0.2",
     "psr/http-client": "^1.0.1",
     "psr/http-message": "^1.0",
-    "rakit/validation": "^1.4"
+    "rakit/validation": "^1.4",
+    "ext-json": "*",
```

Comment:
Keep these in alphabetical order.

Reply:
Worth throwing `sort-packages` into your Composer config to do this for you:

https://getcomposer.org/doc/06-config.md#sort-packages

Reply:
Adding `sort-packages` to the composer config and re-alphabetized the dependencies.